### DISCUSSION TITLE
Tool to help create datasets for this trainer, and some samples of training

### DISCUSSION THREAD

**Discussion by devilismyfriend** [152334H/DL-Art-School#13] [Show and tell]
Hey,
I wrote a small tool to produce a compatible dataset for this tool, you can test it out here:
https://github.com/devilismyfriend/ozen-toolkit

It will segment, transcribe and save audio in the LJ format, can be run on a single or a folder of files.

I also started experimenting with training using this tool.
This is the Elder God from Legacy of Kain, fine-tuned for 200 epochs, 8 total sound files, all between 2s to 20s, 6 for training 2 for validation, lots of reverb in the audio.

OG Tortoise
https://vocaroo.com/149RfG7Y146o

Finetuned Tortoise (same seed)
https://vocaroo.com/14wekiD7FnE4

Training sample example
https://vocaroo.com/1hJ6q5183Zvn

11labs comparison
https://vocaroo.com/1aJj3ZGO3AvL

Next I fine-tuned Father Gascoigne from Bloodborne, around 20 voice files of similar lengths, 0.2 validation split, and modified settings further.
I don't have a comparison for this one but here a sample 
https://vocaroo.com/1aJj3ZGO3AvL

I think next I'll work on this trainer a bit to make it more convenient to train as I feel it's too hard to currently iterate

**Comment by vdruts** [comment]
Thank you for your work! This is excellent.
